src/enemy.js

```diff
@@ -0,0 +1,65 @@
+// enemy class
+class Enemy {
+    // private properties
+    _element = document.createElement('div');
```

Comment:
This line has the error.

Reply:
This error occured because class fields are not currently supported by ESlint, if you delete the top class field you will see that the error occurs for all declared class field properties and methods, so i think we should instead use normal class properties and methods, but there is a workaround using babel-eslint to allow the use of class fields with eslint.

Reply:
Also no sure if these properties need to be private, and if we did want that we could use #propertyName instead which is a class private field to prevent access to them from outside the class [https://developer.mozilla.org/enUS/docs/Web/JavaScript/Reference/Classes/Private_class_fields](url)

Reply:
Ok kai, I'll just fix it. Thanks